.github/workflows/ci.yml

```diff
@@ -25,17 +27,32 @@ jobs:
       - name: Install
         run: npm install
 
-      - name: Install PeerDeps
-        run: npm i fastify zod
-
```

Comment:
This shouldn't be needed, both `zod` and `fastify` are already present inside `devDependencies`:

- https://github.com/turkerdev/fastify-type-provider-zod/blob/f746da2dc212c5252ae701da067f5ad38a5e1ae5/package.json#L61
- https://github.com/turkerdev/fastify-type-provider-zod/blob/f746da2dc212c5252ae701da067f5ad38a5e1ae5/package.json#L68